app/assets/stylesheets/navbar.scss

```diff
@@ -62,7 +62,7 @@
   }
 }
 
-.dropdown-toggle {
+.dropdown-toggle.user-dropdown {
```

Comment:
Use sass syntax. The "user-dropdown" class should be a subobject. Also prefix this class with "navbar"

Reply:
@aayush-05 I prefixed the class name with navbar but I don't think it nesting applies here because they are same level classes.

Reply:
Got it! My bad